CHANGELOG.develop

```diff
@@ -612,8 +624,10 @@ is enabled
   - ~SPC s t D for =spacemacs/helm-dir-do-pt-region-or-symbol=
 - Don't change =helm-bookmarks= keys to early when helm-mode is enabled (thanks
   to bmag)
+- Fix autoloading bug involving =spacemacs/helm-find-files= (thanks to Jack Kamm)
 **** HTML
 - Added impatient-mode under ~SPC m i~ (thanks to geo7)
+- XML files now open with web-mode by default (thanks to Jordan Gwyn)
```

Comment:
Open XML files with web-mode by default (thanks to Jordan Gwyn)

consistent with summarizing  (start with verb)?

Reply:
Thanks for the suggestion, but from what I gathered from looking at other messages, there are different kinds of changes:  

-   `Subject: Contributor` did something to the code (added key bindings, fixed something)  
    -   Looks like the imperative is usually used in this case: "Add key bindings"
-   `Subject: Some object in the code` now does something different from before  
    -   Message usually in the indicative form: "Key bindings were reorganized", "prefix is now X", "Settings now go here"

Reply:
...but of course it's not consistent at all across the entire changelog, and I don't think it's important to really standardize style or grammar in this context as long as the meaning gets across :)

Reply:
Note, it does not have to be perfect, otherwise we'd be stuck tweaking the change log for months.